au4/src/playback/internal/playbackcontroller.cpp

```diff
@@ -179,78 +175,62 @@ void PlaybackController::togglePlay()
 
 void PlaybackController::play()
 {
-    IF_ASSERT_FAILED(au3Playback()) {
+    IF_ASSERT_FAILED(player()) {
         return;
     }
 
     if (isLoopEnabled()) {
-        msecs_t startMsecs = playbackStartMsecs();
-        seek(startMsecs);
+        // msecs_t startMsecs = playbackStartMsecs();
+        // seek(startMsecs);
     }
 
-    au3Playback()->play();
+    player()->play();
     setCurrentPlaybackStatus(PlaybackStatus::Running);
 }
 
-void PlaybackController::rewindToStart(const ActionData& args)
+void PlaybackController::rewindToStart(const ActionData&)
 {
     //! NOTE: In Audacity 3 we can't rewind while playing
     stop();
 
-    msecs_t startMsecs = playbackStartMsecs();
-    msecs_t endMsecs = playbackEndMsecs();
-    msecs_t newPosition = !args.empty() ? args.arg<msecs_t>(0) : 0;
-    newPosition = std::clamp(newPosition, startMsecs, endMsecs);
+    // msecs_t startMsecs = playbackStartMsecs();
+    // msecs_t endMsecs = playbackEndMsecs();
+    // msecs_t newPosition = !args.empty() ? args.arg<msecs_t>(0) : 0;
+    // newPosition = std::clamp(newPosition, startMsecs, endMsecs);
 
-    seek(newPosition);
+    // seek(newPosition);
```

Comment:
can you please restore this?

Reply:
The method is called rewindToStart, but we accept an optional position, should we change the name of the method or remove the optional position? In what cases can a position be put?

Reply:
let's remove the optional position

Reply:
fixed